pkg/vere/io/http.c

```diff
@@ -1462,6 +1533,42 @@ _http_req_prepare(h2o_req_t* rec_u,
   return seq_u;
 }
 
+/* _http_spin_accept(): handle incoming http request on spin stack endpoint
+*/
+static c3_i
+_http_spin_accept(h2o_handler_t* han_u, h2o_req_t* rec_u)
+{
+  u3_hcon* hon_u = _http_rec_sock(rec_u);
+  c3_o     aut_o = _http_req_is_auth(&hon_u->htp_u->htd_u->fig_u, rec_u);
+
```

Comment:
We should produce a 404 if there's no spin stack (`!htd_u->stk_u`).

Reply:
Done